f5-bigip-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,83 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'f5-bigip-alerts',
+        rules: [
+          {
+            alert: 'F5BigIPLowNodeAvailabilityStatus',
+            expr: |||
+              100 * (sum(bigip_node_status_availability_state) / clamp_min(count(bigip_node_status_availability_state), 1)) < %(alertsCriticalNodeAvailability)s
```

Comment:
is availability state only a 0 or 1? i thought nodes could have more than 2 states?

Reply:
Only 0 or 1.